license_test.go

```diff
@@ -79,7 +79,7 @@ func TestNewFromDir(t *testing.T) {
 		t.Fatalf("expected error loading empty directory")
 	}
 
-	fPath := filepath.Join(d, "LICENSE")
+	fPath := filepath.Join(d, "License")
```

Comment:
this one actually tests end-to-end mixed case.. however now that we are switching to all-lower-case, the original test will work. I will revert.